### DISCUSSION TITLE
check which version of nginx-proxy

### DISCUSSION THREAD

**Discussion by zanzara** [nginx-proxy/nginx-proxy#1752] [Q&A] [answered]
How can I check which version of nginx-proxy I'm runnig?
Is there a simple cmd?
`docker exec reverse-proxy  .... `

And how can I update easily to the latest image here?
`docker-compose build ...` 

Appreciating any running example. Thx.

**Comment by buchdag** [comment, marked as answer]
> How can I check which version of nginx-proxy I'm runnig?
> Is there a simple cmd?

Unfortunately there is currently no way to check the version of a running nginx-proxy as it has no knowledge of its own version.

I'll try to add that shortly.

> And how can I update easily to the latest image here?

The easiest and most stable way is to use tagged versions in your compose file and to update them when a new semver tag is released. We push minor versions as well as patch versions tags. There isn't any built-in self update process and this isn't a planned feature.

As discussed in #1648, please don't use `latest` in production. If you require a feature that is in `latest` but not yet in a tagged release, consider making you own tagged version from `latest` using the corresponding commit of the `main` branch at the time of image pull:

```console
docker pull nginxproxy/nginx-proxy:latest
docker tag nginxproxy/nginx-proxy:latest nginxproxy/nginx-proxy:7bb3fd5
```

This will prevent unwanted update to a newer `latest` image and allow you to keep track of the actual version of your image.
